### DISCUSSION TITLE
Cannot read .su2 multizone files

### DISCUSSION THREAD

**Discussion by HANSIeltsKing** [su2code/SU2#1599] [Q&A]
When I was applying a multizone simulation with a single physics, I encountered the following problem:
When MULTIZONE_MESH = YES, the merged xxx.su2 file cannot be read. I didn't find any formatting errors also marked IZONE=1 and IZONE=2. The details of the problem are as follows:

```
------------------- Geometry Preprocessing ( Zone 0 ) -------------------
Error in "void CSU2ASCIIMeshReaderFVM::ReadMetadata()":
-------------------------------------------------- -----------------------
Could not find the IZONE= keyword or the zone contents.
 Check the SU2 ASCII file format.
------------------------------ Error Exit ------------------ -------------
```

When MULTIZONE_MESH = NO, I don't know if it is the problem of file reading again. An unexpected failure occurred when the solution process reached the following step:

```
---------------------------------------- Begin Solver ------------------ -------------
Simulation Run using the Multizone Driver
The simulation will run for 100 time steps.
 Performing rigid mesh transformation.
 New physical time: 0 seconds.
 Translational velocity: (0, 0, 0) m/s.
 Plunging frequency: (0, 0, 0) rad/s.
 Plunging amplitude: (0, 0, 0) m.
 Pitching frequency: (0, 0, 62.8) rad/s.
 Pitching amplitude: (0, 0, 30) degrees.
 Pitching phase lag: (0, 0, 0) degrees.
 Angular velocity: (0, 0, 0) rad/s.
Segmentation fault (core dumped)
```

I followed every step of the tutorial exactly, but still can't achieve multizone simulation. 

 - Ubuntu: 20.04
 - SU2 Version: 7.0.4 Linux Version

**Comment by pcarruscag** [comment]
If you don't provide the files for us to replicate your problem the best I can tell you is to look at TestCases/sliding_interface/supersonic_vortex_shedding
